network/websocket.ksy

```diff
@@ -0,0 +1,77 @@
+meta:
+  id: websocket_dataframe
+  title: WebSocket
+  xref:
+    rfc: 6455
+  endian: be
+doc: |
+  The WebSocket Protocol enables two-way communication between a client
+  running untrusted code in a controlled environment to a remote host
+  that has opted-in to communications from that code.  The security
+  model used for this is the origin-based security model commonly used
+  by web browsers.  The protocol consists of an opening handshake
+  followed by basic message framing, layered over TCP.  The goal of
+  this technology is to provide a mechanism for browser-based
+  applications that need two-way communication with servers that does
+  not rely on opening multiple HTTP connections (e.g., using
+  XMLHttpRequest or <iframe>s and long polling).
```

Comment:
This is a copy-paste of copyrighted text from IETF standard: https://tools.ietf.org/id/draft-ietf-hybi-thewebsocketprotocol-09.html

Unfortunately, it's "All rights reserved", so you can't just copy-paste it here.